andela/travel_tool_front
# #160710730 Add maintainance reason

#### What does this PR do?
Add the modal to add maintenance reason and duration
Add timeline of the maintenance on the calendar

#### Description of Task to be completed?
- Add modal for the maintenance reason
- Add timeline on the calendar

#### How should this be manually tested?
- Git clone the front-end repository
- Install dependencies `yarn install`
- checkout to `ft-room-maintainance-reason-#160710730`
- Run the following backend `https://github.com/andela/travel_tool_back/pull/114`.
- Running tests `yarn test`
- Login to the application.
- Change your role to that of `travel administrator`
- Navigate to `http://travel-tool.andela.com:3000/residence/manage/guest-houses/guest-house-1`
- Click the ellipses. A checkbox appears.
- Click the checkbox.
- Modal appears. Add the reason

#### Any background context you want to provide?
- You have to be a travel admin

#### What are the relevant pivotal tracker stories?
[#160710730](https://www.pivotaltracker.com/story/show/160710730)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-18 at 21 09 58" src="https://user-images.githubusercontent.com/23402614/47175197-37e2fa80-d31b-11e8-9da6-b075709b5706.png">
<img width="1204" alt="screen shot 2018-10-18 at 21 10 18" src="https://user-images.githubusercontent.com/23402614/47175198-37e2fa80-d31b-11e8-8392-2d01c9750ced.png">

#### Questions:
None
